### ISSUE TITLE
Import Collada files (dae)

### ISSUE THREAD

**Issue by jdberwong** [helix-toolkit/helix-toolkit#130]
Hi,

Congrats for the Helix-toolkit, is really good.

I am looking forward to import a dae file, because all the features that have collada (Animations, materials, textures, lights) and i would use it in my aplication with your toolkit; the only way i found it, is mixing my sln with others references like ab3d or ASSIMPNET to import the model. The bad thing of this is that the model cant be modified, transform, scale, rotate, or execute any animation contained in the file.

I think that the way to solve this is using ASSIMPNET to import the file, am trying to do the match between Helix (Model3D) and ASSIMPTNET (it return a Scene) but i have no too much experience to do this.

Could you add a importer of dae files? or any suggestions to do this?

**Comment by objorke** [comment]
Yes, it would be nice to have an interface that let you use assimp to import a file to WPF. I don't want to  add a reference into the HelixToolkit, HelixToolkit.Wpf or HelixToolkit.Wpf.SharpDX projects, but it would be great to see a HelixToolkit.Assimp that can be used to import to both Wpf and Wpf.SharpDX models!

**Comment by objorke** [comment]
Assimp has moved to Github, right?
- https://github.com/assimp/assimp/
- https://github.com/assimp/assimp/tree/master/port/Assimp.NET

**Comment by jdberwong** [comment]
It would be great to have HelixToolkit.Assimp.

The first link for assimp (c++) is correct, but i think that the another one is https://code.google.com/p/assimp-net/downloads/list?can=1&q= in the version 3.2, but it seems that is deprecated.

Check this link, is the lib AssimpNet used in the open3dmod(Viewer) app, it seems newer than others. https://github.com/acgessler/open3mod/tree/master/libs/assimp-net . This Viewer app is recommended by the assimp team in their website.

**Comment by objorke** [comment]
Thanks for the links. The location of the Assimp project is a bit confusing, search results shows sourceforge, google code and github...

But I guess the most important link is the nuget package. Is this the correct one:
https://www.nuget.org/packages/AssimpNet/

**Comment by jdberwong** [comment]
Thanks for your help.

Please let me know if I could help you to make HelixToolkit.Assimp Posible!

**Comment by objorke** [comment]
Yes, of course we would like you to contribute :-) Please create a fork! I can try to give some input on the architecture. I think one requirement should be that the same Assimp library should be used for different platforms.

**Comment by jdberwong** [comment]
According this link, i think the correct one is google code link.
https://github.com/acgessler/open3mod/issues/21#event-266280852

https://code.google.com/p/assimp-net/downloads/list?can=1&q=
